demisto_sdk/commands/changelog/changelog.py

```diff
@@ -0,0 +1,251 @@
+    def __init__(
+        self, pr_number: str, pr_name: str = "", release_version: str = None
+    ) -> None:
+        self.pr_number = pr_number
+        self.pr_name = pr_name
+        self.handle_error = ""
+
+    """ VALIDATE """
+
+    def validate(self) -> bool:
+        """
+        ...
+        """
+        if self.is_release():
+            if not self.is_log_folder_empty():
+                logger.error(
+                    "Logs folder is not empty,\n"
+                    "It is not possible to release until the `changelog.md` "
+                    "file is updated, and the `.changelog` folder is empty"
+                )
+                return False
+            if not self.is_changelog_changed():
+                logger.error(
+                    "The file `changelog.md` is not updated\n"
+                    "It is not possible to release until the `changelog.md` "
+                    "file is updated, and the `.changelog` folder is empty"
+                )
+                return False
+        else:
+            if self.is_changelog_changed():
+                logger.error(
+                    "It is not possible to make a change in the `changelog.md` file,\n"
+                    "run `demisto-sdk changelog --init -pn <pr number> -pt <pr name>` to create log file,\n"
+                    "and put the description of your changes there"
+                )
+                return False
+            if not self.is_log_yml_exist():
+                logger.error(
+                    "There is no log file describing your change,\n"
+                    "Please run `demisto-sdk changelog --init -pn <pr number> -pt <pr name>` to create one, "
+                    "and put the description of your changes there"
+                )
+                return False
+            if not self.validate_log_yml():
+                return False
+
+        return True
+
+    """ INIT """
+
+    def init(self, breaking: str = None, feature: str = None, fix: str = None) -> None:
+        """
+        Creates a new log file for the current PR
+        - if `fix` or `breaking` or `feature` is provided
+          it will create the file with the provided arguments
+        - if no argument is provided, it will create the file with initial values,
+          then the user has to change the values manually
+        """
+        if self.is_release():
+            logger.error(
+                "This PR is for release, please use in changelog release command"
+            )
+
+        msg = (
+            f"The creation of the log file .changelog/{self.pr_number}.yml is complete"
+        )
+
+        if breaking or feature or fix:
+            log = self.create_log(breaking, feature, fix)
+        else:
+            msg += ",\nPlease go to the file and edit the initial values."
+            log = INITIAL_LOG
+
+        with (CHANGELOG_FOLDER / f"{self.pr_number}.yml").open("w") as f:
+            yaml.dump(log, f)
+
+        logger.info(msg)
```

Comment:
instead of saving `msg`, better call the log in the code. This makes it better readable (and helps maintainers tell what is going on, like a comment)